css/_vertical_filmstrip_overrides.scss

```diff
@@ -161,7 +172,7 @@
     .video-state-indicator.moveToCorner {
         transition: right 0.5s;
 
-        &.with-filmstrip.with-remote-videos {
```

Comment:
The greatest danger I see with comments: getting outdated. The comments about the with-remote-videos and without-remove-videos classes can be deleted too.